.travis.yml

```
@@ -18,6 +18,7 @@ script:
 - tox
 - tox -e flake8
 - pylint *.py resources/lib/ test/
+- make check-translations
```

I would leave the low-level command in.
In theory we could run make test in .travis, but I like to have both independently run all tests.

---

Is there a reason for this? I normally setup a bunch of `check-*` commands in my makefile that can be run during development, and one `make check` that is ran during ci (and can also be run during development, as one final check).

---

If you run `make` it either stops at the first error, or it ignore errors.
Travis runs all tests, even if one of the tests fails.
So you get to see all encountered errors for each test rather than the first error.

---

It saves a lot of time.

---

I see, but we can still use all the different make check-* commands in travis, so we don't need to modify our checks in two locations. And they still will all be executed. I will create a new issue for this.

---

You could indeed.